dashbuilder-client/dashbuilder-dataset-editor/src/main/java/org/dashbuilder/dataset/editor/client/screens/DataSetDefWizardScreen.java

```diff
@@ -216,10 +216,10 @@ void onSave(final DataSetDef dataSetDef, final String message) {
 
     RemoteCallback<Path> saveSuccessCallback = new RemoteCallback<Path>() {
         @Override public void callback(Path path) {
+            currentWorkflow.clear();
```

Comment:
@dgutierr I'm really curious why this wasn't causing fail in Dashbuilder showcase, only in kie-wb. Any explanation for that? :)